{
  testSame("try {" + "  throw Error();" + "} catch (e) {"+ "  console.log(e.name)"+ "}");
}
